packages/rule-components/src/RuleTable/index.scss

```diff
@@ -4,11 +4,30 @@
 @import '~@redhat-cloud-services/frontend-components/components/PrimaryToolbar.css';
 
 .rhcs-c-rules-table {
-    .ins-c-table__toolbar.ins-m-footer .pf-c-pagination.pf-m-footer {
-        margin-left: auto;
+  .ins-c-table__toolbar.ins-m-footer .pf-c-pagination.pf-m-footer {
+    margin-left: auto;
+  }
+
+  .pf-c-select__menu .pf-c-check__label .ins-battery svg {
+    height: 1.25rem;
```

Comment:
You should use `--pf-global--FontSize--xl`

Reply:
fixed